runtime/vm/monhelpers.c

```diff
@@ -257,7 +257,7 @@ objectMonitorInflate(J9VMThread* vmStruct, j9object_t object, UDATA lock)
 	((J9ThreadAbstractMonitor*)monitor)->count = J9_FLATLOCK_COUNT(lock);	
 
 	if (!LN_HAS_LOCKWORD(vmStruct,object)) {
-		objectMonitor->alternateLockword = (j9objectmonitor_t)((UDATA)objectMonitor | OBJECT_HEADER_LOCK_INFLATED);
+		J9_STORE_LOCKWORD(vmStruct, &objectMonitor->alternateLockword, (j9objectmonitor_t)((UDATA)objectMonitor | OBJECT_HEADER_LOCK_INFLATED));
```

Comment:
I assume this is required because of the use of `J9_LOAD_LOCKWORD...` macros? 
Like: https://github.com/eclipse/openj9/blob/51d9fc6a3da153f5b22a30dce3dc661574f5acce/runtime/util/moninfo.c#L70

Reply:
Yes, the lockword location is treated as if it were in the object, so the load/store macros must always be used. Again, this works on X because of endian.